.azure-pipelines.yml

```diff
@@ -83,3 +83,44 @@ jobs:
       testResultsFiles: '**/test-results.xml'
       testRunTitle: 'Python $(python.version)'
     condition: succeededOrFailed()
+
+- job: 'TestWindows'
+  displayName: 'Test Win2019'
+  condition: "succeeded()"
```

Comment:
pretty sure this is a dead line on both the linux matrix and windows matrix, as they don't have a direct dependency so execute in parallel